pkg/rancher-components/src/components/Form/LabeledInput/LabeledInput.vue

```diff
@@ -144,11 +144,16 @@ export default (
       if (this.type !== 'cron' || !this.value) {
         return;
       }
-      if (!isValidCron(this.value)) {
```

Comment:
We should port this fix in rancher/dashboard.

Reply:
will have PR to add in rancher/dashboard.